terraform/stacks/external/stack.tf

```diff
@@ -1,5 +1,5 @@
 module "cdn_broker" {
```

Comment:
Yes, there would be two modules included in this stack. One for the user, one for the bucket.